formal/promela/src/testbuilder.py

```diff
@@ -99,12 +112,13 @@ def ready_to_generate(model):
     return ready
 
 
+@catch_subprocess_errors
 def generate_spin_files(model, spinallscenarios):
     """Create spin files from model"""
     if not ready_to_generate(model):
         sys.exit(1)
     print(f"Generating spin files for {model}")
-    subprocess.run(f"spin {spinallscenarios} {model}.pml",
+    subprocess.run(f"spin {spinallscenarios} {model}.ml",
```

Comment:
This is not the best way to test user error handling. This is a code error. The issue here is  you didn't undo this in the next commit used for the PR